CodeMaid/StringResourceKey.cs

```diff
@@ -0,0 +1,34 @@
+﻿using SteveCadwallader.CodeMaid.Integration.Commands;
+using System;
+using System.Collections.Specialized;
+using System.Linq;
+
+namespace SteveCadwallader.CodeMaid
+{
+    internal class StringResourceKey : Properties.Resources
```

Comment:
I think I know what's going on here, but can you explain this file to me to make sure?

Reply:
Stringresourcekey is the name of the string resource class that is used by default when Tom-englert/resxresourcemanager is automatically generated. It's not perfect, but it's the best localized translation tool I've ever seen. Use of this file is still to cater to the tool to make the translation faster completion.

Reply:
Ok, but can you explain what this file is doing to me?

Reply:
Using resources requires referencing namespace SteveCadwallader.CodeMaid.Properties, but using Stringresourcekey does Not require Stevecadwallader.codemaid, because the root namespace is it. It would be awkward to add namespaces to all required code SteveCadwallader.CodeMaid.Properties.

Reply:
It looks like this class isn't doing anything at all anymore, so we could delete it and have everything point consistently to Resources.  I know it means adding a using statement to do those C# files, but I do not think that is a bad thing and I think it's simpler than having two seeming versions of the resources.